hummingbot/strategy/fixed_grid/fixed_grid.pyx

```diff
@@ -0,0 +1,934 @@
+        self._logging_options = logging_options
+
+
+    def grid_assets_df(self) -> pd.DataFrame:
+        market, trading_pair, base_asset, quote_asset = self._market_info
+        price = self._market_info.get_mid_price()
+        base_balance = float(market.get_balance(base_asset))
+        quote_balance = float(market.get_balance(quote_asset))
+        available_base_balance = float(market.get_available_balance(base_asset))
+        available_quote_balance = float(market.get_available_balance(quote_asset))
+        base_value = base_balance * float(price)
+        total_in_quote = base_value + quote_balance
+        base_ratio = base_value / total_in_quote if total_in_quote > 0 else 0
+        quote_ratio = quote_balance / total_in_quote if total_in_quote > 0 else 0
+        data=[
+            ["", base_asset, quote_asset],
+            ["Total Balance", round(base_balance, 4), round(quote_balance, 4)],
+            ["Available Balance", round(available_base_balance, 4), round(available_quote_balance, 4)],
+            [f"Current Value ({quote_asset})", round(base_value, 4), round(quote_balance, 4)]
+        ]
+        data.append(["Current %", f"{base_ratio:.1%}", f"{quote_ratio:.1%}"])
+        df = pd.DataFrame(data=data)
+        return df
+
+    def active_orders_df(self) -> pd.DataFrame:
+        market, trading_pair, base_asset, quote_asset = self._market_info
+        price = self.get_mid_price()
+        active_orders = self.active_orders
+        no_sells = len([o for o in active_orders if not o.is_buy and o.client_order_id])
+        active_orders.sort(key=lambda x: x.price, reverse=True)
+        columns = ["Level", "Type", "Price", "Spread", "Amount (Orig)", "Amount (Adj)", "Age"]
+        data = []
+        lvl_buy, lvl_sell = 0, 0
+        for idx in range(0, len(active_orders)):
+            order = active_orders[idx]
+  
+            if order.is_buy:
+                level = lvl_buy + 1
+                lvl_buy += 1
+            else:
+                level = no_sells - lvl_sell
+                lvl_sell += 1
+            spread = 0 if price == 0 else abs(order.price - price)/price
+            age = "n/a"
+            # // indicates order is a paper order so 'n/a'. For real orders, calculate age.
+            if "//" not in order.client_order_id:
+                age = pd.Timestamp(int(time.time()) - int(order.client_order_id[-16:])/1e6,
+                                   unit='s').strftime('%H:%M:%S')
+
+           
+            amount_orig = ""
+
+            data.append([
+                level,
+                "buy" if order.is_buy else "sell",
+                float(order.price),
+                f"{spread:.2%}",
+                amount_orig,
+                float(order.quantity),
+                age
+            ])
+
+        return pd.DataFrame(data=data, columns=columns)
+
+    def market_status_data_frame(self, market_trading_pair_tuples: List[MarketTradingPairTuple]) -> pd.DataFrame:
```

Comment:
Is there any difference between the strategy_base market_status_dataframe and this market_status_data_frame?